HoloRepositoryDemoApplication/Assets/HoloRpository/Scripts/ModelSetting.cs

```diff
@@ -0,0 +1,86 @@
+﻿using Microsoft.MixedReality.Toolkit.UI;
+using UnityEngine.SceneManagement;
+using UnityEngine;
+using System;
+
+namespace HoloRepository
+{
+    public class ModelSetting : MonoBehaviour
+    {
+        #region Properties
+        private static string ModelName = "LoadedModel";
+        private static Vector3 ModelPosition = new Vector3(0f, 0f, 0f);
+        private static Vector3 ModelRotation = new Vector3(0f, 0f, 0f);
+        private static float ModelSize = 1f;
+        private static bool Manipulable = true;
+        private static string SceneName = null;
+
+        #endregion Properties
+
+        #region Set Method
+        public static void SetModelName(string name)
+        {
+            ModelName = name;
+        }
+
+        public static void SetPostition(Vector3 position)
+        {
+            ModelPosition = position;
+        }
+
+        public static void SetRotation(Vector3 rotation)
+        {
+            ModelRotation = rotation;
+        }
+
+        public static void SetSize(float size)
+        {
+            ModelSize = size;
+        }
+
+        public static void SetManipulable(bool manipulable)
+        {
+            Manipulable = manipulable;
+        }
+        public static void SetSeceneName(string scenename)
```

Comment:
As I mentioned before, Scene switch in HoloLens app is different with normal unity project, It reqiures a base scene which is always on, so initially, the object will be added to the base scene, and it will not disapper when you change the scenes, So this method is used to determine which scene you want to load the object